stream/common/src/test/java/org/apache/bookkeeper/common/coder/TestByteArrayCoder.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.bookkeeper.common.coder;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+import java.util.Arrays;
+import java.util.List;
+import org.junit.Test;
+
+/**
+ * Unit tests for {@link ByteArrayCoder}.
+ */
+public class TestByteArrayCoder extends CoderBasicTestCase {
+
+    private static final ByteArrayCoder TEST_CODER = ByteArrayCoder.of();
+
+    private static final List<byte[]> TEST_VALUES = Arrays.asList(
+        new byte[]{0xa, 0xb, 0xc},
+        new byte[]{0xd, 0x3},
+        new byte[]{0xd, 0xe},
+        new byte[]{});
+
+    @Test(timeout = 10000)
```

Comment:
Are we using Timeout in these tests?

Reply:
these are code made a very long time ago, so there were still `@Test` annotations. will remove them.

Reply:
removed